### PR TITLE
new storyteller: the Lookout

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

Adds a new storyteller: The Lookout.

## Why It's Good For The Game

add: Adds The Lookout, a storyteller that will spawn antagonists and then go silent to let the crew make their own drama. Needs playtesting.

## Changelog
```changelog
add: Adds The Lookout, a storyteller that will spawn antagonists and then go silent to let the crew make their own drama. Needs playtesting.
```

### CODE REVIEW

**Review by Torque4607 — APPROVED** [review]
The code looks good, just needs approval admin-side to merge.

**Comment by bearrrrrrrr** [comment]
SPEEEEEEEEDMEEEEEEEEEEEEEEEEEERGE

**Comment by Torque4607** [comment]
This is basically Bay's `secret` gamemode.

File: zzz_modular_syzygy/storytellers/lookout.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/datum/storyteller/tyrant  //This objective of this storyteller is to spawn antagonists at round-start, and then be quiet to let the crew roleplay and create their own drama.
```
Comment (Michiyamenotehifunana) [inline zzz_modular_syzygy/storytellers/lookout.dm:1]: You forgot to rename the typepath

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]

**Comment by Torque4607** [comment]
Hurk, guess I'm just blind.

File: zzz_modular_syzygy/storytellers/lookout.dm (hunk 1 above)
Comment (bearrrrrrrr) [reply zzz_modular_syzygy/storytellers/lookout.dm]: done

**Review by Michiyamenotehifunana — APPROVED** [review]
Looking good.

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
Wait is this actually included in the .dme? Is The Lookout even going to be compiled? Did you even test this bear why

**Comment by bearrrrrrrr** [comment]
should be good now.

**Review by Michiyamenotehifunana — APPROVED** [review]
can confirm, is good now

**Pull request merged**